nextcord/channel.py

```diff
@@ -689,6 +689,10 @@ async def create_thread(
 
         .. versionadded:: 2.0
 
+        .. note::
+
```

Comment:
Unneeded blank line?

Reply:
No it is not, see https://github.com/nextcord/nextcord/blob/master/nextcord/guild.py#L188-L190 for example